src/server/server.js

```diff
@@ -124,6 +125,16 @@ function processPluginHtml(html, pluginId) {
     });
 }
 
+function stop() {
```

Comment:
We should add this function as an export to `src/simulate.js` as well to expose it to npm.